pkg/tracer/offsetguess.go

```diff
@@ -370,12 +377,18 @@ func guess(b *elf.Module) error {
 		family: syscall.AF_INET,
 	}
 
+	// if the kretprobe for tcp_v4_connect() is configured with a too-low
+	// maxactive, some kretprobe might be missing. In this case, we detect
```

Comment:
Unfortunately, it does not seem possible from userspace. Kernel modules can setup a kretprobe with a different maxactive, but this does not seem to be exposed to userspace.

I've tested a patch to make it configurable (https://github.com/iovisor/bcc/issues/1072#issuecomment-289606421) but I don't have ideas for current kernels.